demux/demux_mkv.c

```diff
@@ -1831,9 +1831,16 @@ static const char *const mkv_sub_tag[][2] = {
     { "D_WEBVTT/CAPTIONS",  "webvtt-webm"},
     { "S_TEXT/WEBVTT",      "webvtt"},
     { "S_DVBSUB",           "dvb_subtitle"},
+    { "S_ARIBSUB",          "arib_caption"},
     {0}
 };
 
+static void avcodec_par_destructor(void *p)
+{
+    if (p)
```

Comment:
Destructor only gets registered when after the talloc, so probably this nullptr check is not required.